ser316asu-2018/Ahrensburg
# US33

Request for Merge into devtest.  Completed tasks 72 & 85.
Test cases:
- Add a new sprint, ensure the checkbox for end-date is removed. 
- Adding/Modifying a sprint should require both a start-date and end-date.
- Start-date through end-date should not overlap between sprints in a project.
